runner/materialize.go

```diff
@@ -15,8 +15,9 @@ import (
 	"github.com/featureform/types"
 )
 
-const MAXIMUM_CHUNK_ROWS int64 = 1024
-const WORKER_IMAGE string = "featureformcom/worker"
+const MAXIMUM_CHUNK_ROWS int64 = 102400
+
+var WORKER_IMAGE string = helpers.GetEnv("WORKER_IMAGE", "featureformcom/worker:0.2.0-rc")
```

Comment:
does it ever pull the image from the environment variable? is there a way to pull the latest? I don't think it is a good idea to hardcode version `0.2.0-rc`

Reply:
changed to latest